packages/cozy-intent/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cozy-intent",
   "description": "Event-based library allowing interactions between React-Native and React applications",
-  "version": "1.13.5",
+  "version": "1.14.0",
```

Comment:
that should be a fix(), in order to make the CI generate a v1.14.1 that fixes version